app/models/author.rb

```diff
@@ -0,0 +1,5 @@
+class Author < ApplicationRecord
+  validates :username, length: { minimum: 4, maximum: 12 }
```

Comment:
- [ ] We still need posts and comments association for the user. Suppose that we need all the posts or the comments for a specific user!
We can add the relevent association with `has_many :posts`. Similarly for `comments` -- Let's add them.

Reply:
fix this